index.js

```diff
@@ -1,35 +1,65 @@
-/*!
- * forwarded
- * Copyright(c) 2014 Douglas Christopher Wilson
- * MIT Licensed
- */
-
-/**
- * Module exports.
- */
+'use strict'
 
-module.exports = forwarded
+var Processor = require('./lib/processor')
+var schemas = require('./lib/schemas')
 
 /**
  * Get all addresses in the request, using the `X-Forwarded-For` header.
  *
- * @param {Object} req
+ * @param {http.IncomingMessage} req
  * @api public
  */
 
-function forwarded(req) {
+module.exports = function forwarded (req, options) {
+  options = options || {}
+
+  var opts = {
+    // default to only common + standard
+    // array order matters here
+    schemas: options.schemas || [
```

Comment:
to clarify are you suggesting to keep the current RFC standard as the default option?

also, another option is to ***not offer a default**, leaving it to the implementer to decide what to support.

e.g. 

- `proxy-addr v1.0.8` will be `['xff']` and in next major release can switch to `['xff', 'rfc7239']`
- `express v4.x` in addressing (strongloop/express/pull/2666)  it could be `['xff']` only, next major release can be `['xff', 'rfc7239']` + with addition of a application setting for users to customize.